services/navigation/builtin/explore_test.go

```diff
@@ -16,7 +16,6 @@ import (
 )
 
 func TestExploreMode(t *testing.T) {
-	t.Skip()
```

Comment:
[note] Test was flaky due to speed difference between mac and pi leading to sometimes less than 100 points. At minimum here we just need to test that the loop is working so 2 points should suffice.